lighthouse-core/test/report/v2/renderer/crc-details-renderer-test.js

```diff
@@ -92,37 +92,18 @@ describe('DetailsRenderer', () => {
     const el = CriticalRequestChainRenderer.render(dom, dom.document(), DETAILS);
     const details = el.querySelector('.lh-details');
     const chains = details.querySelectorAll('.crc-node');
-    assert.equal(chains.length, 4, 'generates correct number of chain nodes');
 
-    const div = dom.createElement('div');
-    div.innerHTML = `<span class="crc-node__tree-marker">
-        <!-- fill me -->
-        <span class="tree-marker up-right"></span>
-        <span class="tree-marker right"></span>
-        <span class="tree-marker horiz-down"></span>
-      </span>
-      <span class="crc-node__tree-value">
-        <span class="crc-node__tree-file">/</span>
-        <span class="crc-node__tree-hostname">(example.com)</span>
-        <!-- fill me -->
-      </span>`;
-    assert.ok(chains[0].innerHTML, div.innerHTML);
+    // Main request
+    assert.equal(chains.length, 4, 'generates correct number of chain nodes');
+    assert.equal(chains[0].querySelector('.crc-node__tree-hostname').textContent, '(example.com)');
 
-    div.innerHTML = `<span class="crc-node__tree-marker">
-        <!-- fill me -->
-        <span class="tree-marker"></span>
-        <span class="tree-marker"></span>
-        <span class="tree-marker vert-right"></span>
-        <span class="tree-marker right"></span>
-        <span class="tree-marker right"></span>
-      </span>
-      <span class="crc-node__tree-value">
-        <span class="crc-node__tree-file">/b.js</span>
-        <span class="crc-node__tree-hostname">(example.com)</span>
-        <!-- fill me -->
-        <span class="crc-node__chain-duration"> - 5,000ms, </span>
-        <span class="crc-node__chain-duration">0KB</span>
-      </span>`;
-    assert.ok(chains[1].innerHTML, div.innerHTML);
+    // Children
+    assert.ok(chains[1].querySelector('.crc-node__tree-marker .vert-right'));
+    assert.equal(chains[1].querySelectorAll('.crc-node__tree-marker .right').length, 2);
+    assert.equal(chains[1].querySelector('.crc-node__tree-file').textContent, '/b.js');
+    assert.equal(chains[1].querySelector('.crc-node__tree-hostname').textContent, '(example.com)');
+    const durationNodes = chains[1].querySelectorAll('.crc-node__chain-duration');
+    assert.equal(durationNodes[0].textContent, ' - 5,000ms, ');
+    assert.equal(durationNodes[1].textContent, '1.95KB');
```

Comment:
maybe put a comment/assertion message that this is the `transferSize` and 2000 bytes is being formatted to KB? Took me a little while to see where it came from/if the new transfer sizes were being tested.